### ISSUE TITLE
Use button instead of span for listitems with child menu

### ISSUE THREAD

**Issue by raphtag** [FrDH/jQuery.mmenu#686]
Hi, for a better accessibility it would be better to use a focusable button instead of the span element as a not clickable list item with with a childmenu
Is this is possible in any way?

**Comment by FrDH** [comment]
I guess that would be possible, but why should a list item be focusable is it isn't clickable?

**Comment by raphtag** [comment]
Hi it should be for a better accessibility, for users only using a keyboard.
